apps/showcase/src/services/webcontainer/webcontainer.service.ts

```diff
@@ -126,4 +126,32 @@ export class WebContainerService {
 
     return getFilesTreeFromContainer(instance, EXCLUDED_FILES_OR_DIRECTORY);
   }
+
+  /**
+   * Retrieve all typescript declaration files from web-container
+   * @param project
+   * @param maxDepth
+   * @param path
+   */
+  public async getDeclarationTypes(project: string, maxDepth = 20, path = 'node_modules'): Promise<{ filePath: string; content: string }[]> {
+    // TODO read that from project devDependencies?
```

Comment:
devDependencies can be removed according to the builder (as it is with ng-packagr)

Reply:
Here, I'm talking about the `devDependencies` of the tutorial app inside the web-container